<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoliProperties</class>
 <widget class="QDialog" name="MoliProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>磨粒属性</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1051</width>
     <height>220</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_17">
    <item row="3" column="2">
     <widget class="QComboBox" name="_comboBoxMoliGuy">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="_comboBoxMoliID">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QLabel" name="_labelResultImage">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_121">
      <property name="text">
       <string>磨粒颜色</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_109">
      <property name="text">
       <string>磨粒材质</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_118">
      <property name="text">
       <string>磨粒标注人</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLineEdit" name="_editMoliSize"/>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_122">
      <property name="text">
       <string>磨粒磨损机理</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_113">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;磨粒图片(路径)&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="label_124">
      <property name="text">
       <string>磨粒表面纹理类型</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_116">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;磨粒编号&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="label_131">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="label_123">
      <property name="text">
       <string>磨粒尺寸(微米)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QLineEdit" name="_editMoliLength"/>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="_labelOriginalImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="label_132">
      <property name="text">
       <string>铁谱分析报告编号</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="_comboBoxMoliShape">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_111">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;铁谱图片编号&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="label_112">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;铁谱片编号&lt;span style=&quot; color:#ff0000;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QLabel" name="label_126">
      <property name="text">
       <string>磨粒周长(微米)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="_comboBoxMoliProperty">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_129">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_120">
      <property name="text">
       <string>磨粒位置</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_130">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="_comboBoxMoliImageID">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QLabel" name="_labelMaskImage">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label_110">
      <property name="text">
       <string>磨粒形状</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QComboBox" name="_comboBoxMoliPianID">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLabel" name="label_127">
      <property name="text">
       <string>磨粒磨损类型</string>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QComboBox" name="_comboBoxMoliReportID">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4" colspan="6">
     <widget class="QLineEdit" name="_editMoliPath">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QComboBox" name="_comboBoxMoliColor">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QComboBox" name="_comboBoxMoliErodeReason">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QComboBox" name="_comboBoxMoliPosition">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QComboBox" name="_comboBoxMoliErodeType">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_114">
      <property name="text">
       <string>磨粒损伤部位</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QComboBox" name="_comboBoxMoliSurface">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QComboBox" name="_comboBoxMoliGivenInfo">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="label_125">
      <property name="text">
       <string>磨粒反映故障信息</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QLabel" name="label_128">
      <property name="text">
       <string>磨粒典型性</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QComboBox" name="_comboBoxMoliTypical">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QComboBox" name="_comboBoxMoliErodePart">
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>270</y>
     <width>178</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_133">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>350</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*说明&lt;/span&gt;: 磨粒编号继承自铁谱图片编号, 最后两位是磨粒顺序编号, 默认为00, 后续为01, 02, ..., 不超过99.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
